src/libpmem/pmem.c

```diff
@@ -1278,3 +1294,180 @@ pmem_init(void)
 			"QueryVirtualMemoryInformation");
 #endif
 }
+
+/* XXX: deep flush */
+#define FLUSH_SDS(sds) pmem_flush(sds, sizeof(PMEMsds))
+
+#ifndef _WIN32
```

Comment:
ifdef linux.
But honestly, there's nothing linux-specific here. The purpose of the os_dimm abstraction layer is exactly to separate platform-specific code, and that's where it should be separated.
Just write an empty os_ras_bsd.c and os_ras_windows.c